scripts/globals/mobskills/viscid_emission.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------
+--  Viscid Emission
+--
+--  Inflicts amnesia in an area of effect.
+--  Type: Magical (Enfeebling)
+--  Range: Unknown cone
+--  Utsusemi/Blink absorb: Ignores shadows
+--
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local typeEffect = xi.effect.AMNESIA
+    local duration = 60
+
+    skill:setMsg(MobStatusEffectMove(mob, target, typeEffect, 1, 0, duration))
+
+    return typeEffect
+end
+
+return mobskill_object
```

Comment:
Could you please add a newline to the end of the file?

Reply:
Sure. I'll apply this to my other pull request as well.